ads/criteo.js

```diff
@@ -53,8 +59,60 @@ export function criteo(global, data) {
 /**
  * @param {!Window} global
  * @param {!Object} data
+ * @param {!function(*)} resultCallback
  */
-function setTargeting(global, data) {
+function criteoStandalone(global, data, resultCallback) {
+  const adUnitParams = {
+    'integrationmode': 'amp',
+    'placements': [
+      {
+        'slotid': data.slot,
+        'zoneid': data.zone,
+      },
+    ],
+  };
+  Criteo.RequestBids(adUnitParams, bids => {
+    listenForCreativeRequests();
+    for (const i in bids) {
+      const bid = bids[i];
+      if (data.adserver === 'DFP') {
+        const lir = data.lineItemRanges;
+        const targeting = Criteo.ComputeStandaloneDFPTargeting(bid, lir);
+        setTargeting(global, data, targeting);
+      }
+    }
+    resultCallback(null);
+  }, data.timeout);
+}
+
+function listenForCreativeRequests() {
```

Comment:
Done, I moved all of the standalone logic into the PublisherTag. Is that enough/too much? (note that the PublisherTag version including `Criteo.PubTag.Adapters.AMP.Standalone` is not released yet)

Would another PR to move the "RTA" logic, or, in an even more extreme way, most of the file in PublisherTag be appreciated? (pretty much only leaving the `loadScript` part in AMP) Or is that too much?